oauth-common/src/main/java/io/strimzi/kafka/oauth/validator/JWTSignatureValidator.java

```diff
@@ -59,10 +63,9 @@ public class JWTSignatureValidator implements TokenValidator {
 
     private static final Logger log = LoggerFactory.getLogger(JWTSignatureValidator.class);
 
-    private final DefaultJWSVerifierFactory verifierFactory = new DefaultJWSVerifierFactory();
```

Comment:
I realise this boat has sailed, but why is this class shouty `JWT...`, but we've got `JmxMetrics` etc?

Reply:
Sometimes the shouty version is more readable so I opt for a shouty one.
It would be ideal to have consistency, but then readability is also a factor. Granted it's a subjective criteria, but then a lot of software development is choosing between multiple approaches and weighing pros and cons.